packages/babel-plugin-ember-test-metadata/__tests__/utils-test.js

```diff
@@ -32,111 +29,120 @@ describe('getNodeProperty', () => {
+
+      expect(formattedPath).toBe(expectedPath);
+    });
+
+    it("returns the normalized filepath for the in app test", () => {
+      const expectedPath = "tests/acceptance/foo-test.js";
+      const preFormattedPath = "/Users/tester/workspace/personal/test-bed/classic/classic/tests/acceptance/foo-test.js";
+      const opts = { filename: preFormattedPath, root: "/Users/tester/workspace/personal/test-bed/classic" };
+      const config = {
+        pkg: {
+          name: "classic"
+        },
+      }
+      config.pkg['ember-addon'] = {
+        paths: ['lib/bar']
+      }
+
+      const formattedPath = getNormalizedFilePath(opts, config);
+
+      expect(formattedPath).toBe(expectedPath);
+    });
+
+    it("returns the normalized filepath with the addon name", () => {
+      const expectedPath = `lib/bar/tests/acceptance/foo-test.js`
+      const preFormattedPath = `${appRoot}/tests/ember-add-in-repo-tests/lib/bar/tests/acceptance/foo-test.js`;
+      const opts = { filename: preFormattedPath, root: appRoot };
+      const config = {
+        pkg: {
+          name: "classic"
+        },
+      }
+      config.pkg['ember-addon'] = {
+        paths: ['lib/bar']
+      }
+
+      const formattedPath = getNormalizedFilePath(opts, config);
+
+      expect(formattedPath).toBe(expectedPath);
+    });
   });
-});
 
-describe('_getParsedEmbroiderFilepath', () => {
-  const filename = path.join(
-    'private',
-    'var',
-    'folders',
-    'abcdefg1234',
-    'T',
-    'embroider',
-    '098765',
-    'tests',
-    'acceptance',
-    'my-test.js'
-  );
-  const pathSegments = filename.split(path.sep);
-
-  it('returns file path from classic build correctly', () => {
-    expect(_getParsedEmbroiderFilepath(pathSegments)).toBe(
-      path.join('tests', 'acceptance', 'my-test.js')
-    );
+  describe("given file path from an embroider build", () => {
+    beforeEach(() => {
+      appRoot = '/private/var/folders/abcdefg1234/T/embroider/098765';
+    });
+
+    it("returns the normalized filepath for app tests", () => {
+      const expectedPath = `tests/acceptance/foo-test.js`
+      const preFormattedPath = `${appRoot}/tests/acceptance/foo-test.js`;
+      const config = {
+        pkg: {
+          name: "example-app",
+        },
+      }
+      const opts = { filename: preFormattedPath, root: appRoot };
+
+      const formattedPath = getNormalizedFilePath(opts, config);
+
+      expect(formattedPath).toBe(expectedPath);
+    });
+
+    it("returns the normalized filepath for addon app tests", () => {
+      const expectedPath = `tests/acceptance/foo-test.js`
+      const preFormattedPath = `${appRoot}/tests/acceptance/foo-test.js`;
+      const config = {
+        pkg: {
+          name: "example-app",
+        },
+      }
+      config.pkg['ember-addon'] = {
+        paths: ["lib/bar"]
+      }
+      const opts = { filename: preFormattedPath, root: appRoot };
+
+      const formattedPath = getNormalizedFilePath(opts, config);
```

Comment:
Can we name this `normalizedFilePath`, since you're calling a function called `getNormalizedFilePath`, which is intended to return a `normalizedFilePath`.